src/libs/actions/Policy.ts

```diff
@@ -67,6 +67,13 @@ type OptimisticCustomUnits = {
     outputCurrency: string;
 };
 
+type UpdateCustomUnit = {
+    customUnitID: string;
+    name: string;
+    attributes: Attributes;
+    rates: Rate;
+};
```

Comment:
Also maybe I'd rename it to NewCustomUnit or CustomUnitUpdate
```suggestion
type UpdateCustomUnit = CustomUnit & {
    rates: Rate;
};
```